lib/WebService/Async/Onfido.pm

```diff
@@ -830,6 +835,73 @@ sub report_list {
     return $src;
 }
 
+=head2 countries_list
+
+Returns a hashref containing 2-letter country codes as keys and supporting status
+as their value.
+
+=cut
+
+sub countries_list {
```

Comment:
Could probably cache this, since the values don't change regularly?

Reply:
It's being cached in [RPC](https://github.com/regentmarkets/bom-rpc/pull/2035/files#diff-34235887f3ed9d7f84ca919296355eaeR152) and [Events](https://github.com/regentmarkets/bom-events/pull/75/files#diff-0355a08436e02a46826853bc40525f91R892), as we don't have access to Redis here.